Python/Product/Common/Infrastructure/PathUtils.cs

```diff
@@ -821,5 +821,35 @@ static class PathUtils {
             }
             return string.Compare(filePath, i + 1, ext, (ext[0] == '.' ? 1 : 0), int.MaxValue, StringComparison.OrdinalIgnoreCase) == 0;
         }
+
+        /// <summary>
+        /// Returns the depth (# of subfolders) between two paths
+        /// 
```

Comment:
Just keep that first sentence in the summary (and end it with a dot).
You can put the rest in a <returns> section, and remove the extra indent.